<ui version="4.0">
 <class>PanelProfesor</class>
 <widget class="QMainWindow" name="PanelProfesor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenULE - Panel Profesor</string>
  </property>
  <property name="windowState">
   <set>Qt::WindowFullScreen</set>
  </property>
  <property name="styleSheet">
   <string>
QMainWindow {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b71c1c, stop:1 #ef9a9a);
}
QLabel#labelTitulo {
    color: #b71c1c;
    font-size: 28px;
    font-weight: 900;
}
QLabel {
    font-size: 20px;
    color: white;
    font-weight: bold;
}
QPushButton {
    background-color: #d32f2f;
    color: white;
    font-weight: bold;
    border-radius: 8px;
    padding: 14px 0;
    font-size: 18px;
    min-width: 320px;
    max-width: 340px;
}
QPushButton:hover {
    background-color: #b71c1c;
}
QFrame#cardFrame {
    background: rgba(255,255,255,0.16);
    border-radius: 20px;
    padding: 32px 32px 32px 32px;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="cardFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="cardLayout">
       <item>
        <widget class="QLabel" name="labelTitulo">
         <property name="text">
          <string>
            <span style="font-size:36px;">Bienvenido/a, Mario</span>
          </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <!-- FOTO AÑADIDA -->
       <item>
        <widget class="QLabel" name="labelFoto">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>130</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <!-- FIN FOTO -->
       <item>
        <spacer name="verticalSpacerCardBetweenTitleAndButtons">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="buttonLayout">
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <item>
          <widget class="QPushButton" name="btnVerMenu">
           <property name="text">
            <string>Ver Menú</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConfiguracion">
           <property name="text">
            <string>Configuración</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHistorialReservas">
           <property name="text">
            <string>Historial de Reservas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReportarIncidencia">
           <property name="text">
            <string>Reportar Incidencia</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDarseDeBaja">
           <property name="text">
            <string>Darse de baja</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>